README.md

```diff
@@ -28,7 +28,7 @@ curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-lin
 OR, if you want to avoid using sudo:
 
 ```shell
-curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-linux-amd64 && chmod +x container-diff-linux-amd64 && mkdir $HOME/bin && export PATH=$PATH:$HOME/bin && mv container-diff-linux-amd64 $HOME/bin/container-diff
+curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-linux-amd64 && chmod +x container-diff-linux-amd64 && mkdir -p $HOME/bin && export PATH=$PATH:$HOME/bin && mv container-diff-linux-amd64 $HOME/bin/container-diff
```

Comment:
This change LGTM, but shouldn't we be downloading to /usr/local/bin or something other than `$HOME/bin`, which isn't standard?  We can remove the $PATH modifying too, which won't work for new shells @nkubala

Reply:
there is a snippet for /usr/local/bin installation (via sudo, since user might not have permissions to mess with it as well).
I am curious -- what did you mean by `$PATH modifying ... won't work for new shells`?

Reply:
You need to add `export PATH=$PATH:$HOME/bin` to your bashrc or a similar startup script, otherwise when you open a new shell, the path variable won't include your changes.

Reply:
ah, I just misread your statement, and by "new shells" meant really new types of shells etc, not just new shell sessions.  Thanks for the explanation.
yeah -- in my case PATH already includes ~/bin ;)

Reply:
Yeah this is just a second option if users don't want to use sudo to copy to `/usr/local/bin` or another shared directory.

Reply:
Oh sorry, didn't see that!